src/Nethermind/Nethermind.EthStats.Test/EthStatsClientTests.cs

```diff
@@ -0,0 +1,47 @@
+//  Copyright (c) 2018 Demerzel Solutions Limited
```

Comment:
worth adding the tests to our Code Coverage tool

Reply:
inside .github folder in the repo in the code coverage and test workflows
also need to add the dotcover reference so it can be picked up in the coverage report